pytravharv/__main__.py

```diff
@@ -45,119 +43,49 @@ def get_arg_parser():
         "-n",
         "--name",
         type=str,
+        required=False,
         default=None,
         help="Name of the configuration to use",
     )
 
+    parser.add_argument(
+        "-m",
+        "--mode",
+        choices=["memory", "uristore"],
+        default="memory",
+        required=True,
+        help="Mode to use, either memory or uristore. Default is memory. If memory is used, the target store will be a temporary in-memory store. If uristore is used, the target store will be a URI store.",
+    )
+
     parser.add_argument(
         "-o",
-        "--output-folder",
+        "--output",
         type=str,
-        default=os.path.join(os.getcwd(), "output"),
-        help="Folder to output files to",
+        default=None,
+        required=False,
+        help="File to write output to, if not specified, output will be written to stdout",
+    )
+
+    parser.add_argument(
+        "-c",
+        "--context",
+        nargs="*",
+        required=False,
+        default=None,
+        help="Context to add to graph when asserting paths. This will be a list of Paths to either a file containing triples or a folder containing files that can contain triples.",
     )
 
     parser.add_argument(
         "-ts",
```

Comment:
why is this needed next to the one we have above with `--store` (currently `--mode`) ?